scripts/nginx/autobrr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# autobrr nginx conf
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+users=($(_get_user_list))
+
+if [[ ! -f /etc/nginx/apps/autobrr.conf ]]; then
```

Comment:
I would also not check the conditional here, just write it whether it exists or not